src/cookbooks/glam.md

```diff
@@ -0,0 +1,60 @@
+# Introduction to GLAM
+
+GLAM was built to help Mozillians answer their data questions without needing data analysis or coding skills. GLAM contains a visualization layer meant to answer most “easy” questions about how a probe/metric has changed over build ids and releases.
```

Comment:
```suggestion
GLAM was built to help Mozillians answer their data questions without needing data analysis or coding skills. GLAM contains a visualization layer meant to answer most “easy” questions about how a probe or metric has changed over build ids and releases.
```

Reply:
I use `probe/metric` many times in this article, do you think I should replace all such instances?

Reply:
It's not that big deal, but that seems like a good idea.